lib/lax/messages.ex

```diff
@@ -65,16 +65,42 @@ defmodule Lax.Messages do
     with pid when pid != nil <- GenServer.whereis(:apns_default),
          :direct_message <- channel.type do
       users = Repo.preload(channel, :users).users
+      sender = message.sent_by_user
+      title = "@#{sender.username}"
 
       for user <- users, user.id != message.sent_by_user_id do
         for device_token <- user.apns_device_token do
           bundle_id = "com.example.Lax"
-          message_body = "@#{message.sent_by_user.username}: #{message.text}"
+          subtitle = case Enum.filter(users, &(&1 != user and &1 != sender)) do
```

Comment:
We should use the id of the user structs here for comparison, if there's any difference in the structs themselves i.e. how they're preloaded, then equality will fail when it shouldn't.